source/common/grpc/json_transcoder_filter.cc

```diff
@@ -225,7 +225,7 @@ Http::FilterHeadersStatus JsonTranscoderFilter::decodeHeaders(Http::HeaderMap& h
     readToBuffer(*transcoder_->RequestOutput(), data);
 
     if (data.length() > 0) {
-      decoder_callbacks_->addDecodedData(data);
+      decoder_callbacks_->addDecodedData(data, true);
```

Comment:
There is TODO(lizan) somewhere in here about using watermarks to bound buffer size. I think that is done with this change so we can remove todo? cc @lizan

Reply:
Yeah, I wasn't sure if the transcoder was infinite buffering under the hood, so I figured @lizan could remove the TODO after checking